students/slawinski_amadeusz/lesson_04_unit_testing/collatz_sequence.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import unittest
+
+
+def collatz(number):
```

Comment:
można również zrobić import tej funkcji z lekcji 3:
`from students.slawinski_amadeusz.lesson_03_functions.collatz_sequence collatz`

Reply:
Nie do końca widzę jak to ma działać?

Traceback (most recent call last):
  File "students/slawinski_amadeusz/lesson_04_unit_testing/collatz_sequence.py", line 4, in <module>
    from students.slawinski_amadeusz.lesson_03_functions.collatz_sequence \
ModuleNotFoundError: No module named 'students'

Reply:
To było opcjonalnie. U mnie to akurat działa (PyCharm sam podpowiada ten import), do IDE mam zaimportowane całe repo jako projekt, czasami może tez pomóc dodanie plików __init__.py do folderów z kodem pythona.
Jeżeli dokładnie będę wiedział dlaczego u ciebie nie działa to dam znać.